File: Area/Area.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    double a, b, c;
    cin >> a >> b >> c;
    cout << fixed << setprecision(3) << "TRIANGULO: " << a * c / 2 << endl << "CIRCULO: " << 3.14159 * pow(c, 2) << endl << "TRAPEZIO: " << (a + b) * c / 2 << endl << "QUADRADO: " << pow(b, 2) << endl << "RETANGULO: " << a * b << endl;
}

File: Area/Area.js
const scanner = require("readline");
const data = scanner.createInterface({input: process.stdin, output: process.stdout});
data.question("", (numeros) =>{
    nums = numeros.split(" ");
    console.log(`TRIANGULO: ${(nums[0] * nums[2] / 2).toFixed(3)}`);
    console.log(`CIRCULO: ${(3.14159 * nums[2] * nums[2]).toFixed(3)}`);
    console.log(`TRAPEZIO: ${((nums[0] + nums[1]) * nums[2] / 2).toFixed(3)}`);
    console.log(`QUADRADO: ${(nums[1] * nums[1]).toFixed(3)}`);
    console.log(`RETANGULO: ${(nums[0] * nums[1]).toFixed(3)}`);
    data.close();
});